src/main/java/org/stellar/sdk/responses/operations/PaymentOperationResponse.java

```diff
@@ -64,7 +87,21 @@ public String getFrom() {
     return from;
   }
 
+  public Optional<MuxedAccount> getFromMuxed() {
```

Comment:
'Optional' usage in the interface seems duplicative, `null` can convey same meaning with much less code. Actualy, how do you feel about just converting PaymentOperationResponse to a `lombok` value class instead and avoid all these manuall set/get'rs?

Reply:
We have similar code in [other classes](https://github.com/stellar/java-stellar-sdk/blob/19570b4abd51b11f24b3dd0c9f0881bba39d54e0/src/main/java/org/stellar/sdk/responses/operations/PathPaymentBaseOperationResponse.java#L71-L83), and I want to keep it consistent with them, so I wrote it like this.

Regarding the use of `Optional` or `null` to represent empty values, I believe that using null can indeed make our code look more concise. However, in this case, users must rely on documentation to determine whether a field may be empty. But if `Optional` is introduced, if a field is not set as an `Optional` type, then users can assume that this field definitely exists, which may be more user-friendly.